automation_tools/__init__.py

```diff
@@ -1726,6 +1729,8 @@ def satellite6_upgrade(admin_password=None):
     # Stop katello services, except mongod
     run('katello-service stop')
     run('service-wait mongod start')
+    if major_ver == 7:
+        run('service tomcat stop')
```

Comment:
This is the standard procedure/command we are following in Upgrade procees, which is also documented. So don't want to make any changes in this. So let it be.